android/app/src/main/java/app/organicmaps/downloader/DownloaderNotifier.java

```diff
@@ -29,7 +30,8 @@ public class DownloaderNotifier
   public static final int NOTIFICATION_ID = 1;
 
   private final Context mContext;
-  private NotificationCompat.Builder mProgressNotificationBuilder;
+  private NotificationCompat.Builder mProgressNotificationBuilder = null;
+  private String mNotificationCountryId = null;
```

Comment:
nit: Aren't Java vars null by default?
```suggestion
  private NotificationCompat.Builder mProgressNotificationBuilder;
  private String mNotificationCountryId;
```